src/componets/RadioFormControlLabel/RadioFormControlLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Radio from '@mui/material/Radio';
+
+interface RadioFormControlLabelProps {
+  value: string;
+  label: string;
+  sx: any; 
```

Comment:
using any type in typeScript is considered a bad practice. You can use the `SxProps` type from the mui library